src/redux/actions/exposedActions.js

```diff
@@ -1,5 +1,5 @@
 import core from 'core';
-import isDataElementPanel from 'helpers/isDataElementPanel';
+import isDataElementLeftPanel from 'helpers/isDataElementLeftPanel';
```

Comment:
I changed the name to include `Left`, though the new name is kind of weird... I think we can keep this for now? haha